package.json

```diff
@@ -27,6 +27,7 @@
     "babel-eslint": "^10.0.1",
     "babel-jest": "^23.4.2",
     "babel-loader": "^8.0.4",
+    "babel-plugin-module-resolver": "^4.1.0",
```

Comment:
#nit: prefer non-semantic versions

Reply:
Good call @troygibb! I'm going to merge this PR to unblock my other PR, but I'm going to switch all the packages over this sprint. Tracking here: https://jira.robot.car/browse/AVA-13524